model/src/schema/index.ts

```diff
@@ -1 +1,261 @@
-export { Schema } from "./schema";
```

Comment:
Why are you renaming schema.ts to index.ts?

Reply:
see description

Reply:
Sorry Jen, I can't reproduce the bug you mention and it is weird that index is being used for encapsulation everywhere but fails only in schema, should we investigate better? Is this a production bug?

Reply:
@jenbutongit I've just tried it again and really can't reproduce the bug, both locally and inside doker.

Reply:
@bertrandmc for me, the `rm -rf model/dist` before building fixed it. The problem seemed to be that `yarn model build` didn't overwrite the existing files in `model/dist`